test/functional/mempool_limit.py

```diff
@@ -205,7 +206,7 @@ def test_mid_package_eviction(self):
 
         # Package should be submitted, temporarily exceeding maxmempool, and then evicted.
         with node.assert_debug_log(expected_msgs=["rolling minimum fee bumped"]):
-            assert_raises_rpc_error(-26, "mempool full", node.submitpackage, package_hex)
+            node.submitpackage(package_hex)
```

Comment:
Do you still want to check "mempool full" like below?

Reply:
the check itself is a little weird, so I just double-checked that the package failed. The later checks should check invariants.